server/src/controllers/org_environment_variables.controller.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  Controller,
+  Post,
+  UseGuards,
+  Body,
+  Get,
+  Patch,
+  Delete,
+  Param,
+  BadRequestException,
+  ForbiddenException,
+} from '@nestjs/common';
+import { decamelizeKeys } from 'humps';
+import { JwtAuthGuard } from '../modules/auth/jwt-auth.guard';
+import { PoliciesGuard } from 'src/modules/casl/policies.guard';
+import { User } from 'src/decorators/user.decorator';
+import { EnvironmentVariableDto } from '@dto/environment-variable.dto';
+import { OrgEnvironmentVariablesService } from '@services/org_environment_variables.service';
+import { OrgEnvironmentVariablesAbilityFactory } from 'src/modules/casl/abilities/org-environment-variables-ability.factory';
+import { OrgEnvironmentVariable } from 'src/entities/org_envirnoment_variable.entity';
+
+@Controller('organization-variables')
+export class OrgEnvironmentVariablesController {
+  constructor(
+    private orgEnvironmentVariablesService: OrgEnvironmentVariablesService,
+    private orgEnvironmentVariablesAbilityFactory: OrgEnvironmentVariablesAbilityFactory
+  ) {}
+
+  @UseGuards(JwtAuthGuard)
+  @Get()
+  async get(@User() user) {
+    const result = await this.orgEnvironmentVariablesService.fetchVariables(user);
+    return decamelizeKeys({ variables: result });
```

Comment:
Ignore for fixing decamelizeKeys, I was mistaken that it would decamelize env vars, only the entity attributes would be done for it. Can keep as-is for now.